### PR TITLE
[STAR] add liquid_surfaces_no_lld parameter to aspiration

### PR DESCRIPTION
**Pull request by jkhales**
Allows aspiration while moving the pipette in the z direction

Example of usage:

```
liquid_height = wells[0].compute_height_from_volume(liquid_volume=vol)
liquid_surface = wells[0].get_absolute_location().z + liquid_height

await workcell.lh.aspirate(
    wells,
    vols=[vol],
    liquid_surfaces_no_lld=[liquid_surface],
    surface_following_distance=[liquid_height -1],
    minimum_height=workcell.spin_plate[0][0].get_absolute_location().z
)
```

### CODE REVIEW

File: pylabrobot/liquid_handling/backends/hamilton/STAR.py [hunk 1]
```diff
@@ -1500,6 +1501,8 @@ async def aspirate(
 
       hamilton_liquid_classes: Override the default liquid classes. See
         pylabrobot/liquid_handling/liquid_classes/hamilton/star.py
+      liquid_surface_no_lld: Liquid surface at function without LLD [0.1mm]. Must be between 0
+          and 3600. Default 3600.
```
Comment (rickwierenga) [inline pylabrobot/liquid_handling/backends/hamilton/STAR.py:1505]: commenting for visibility to you: this should be 1mm, between 0 and 360

Comment (jkhales) [reply pylabrobot/liquid_handling/backends/hamilton/STAR.py]: i thought you were requesting changes but seems like you did it 😅

Comment (jkhales) [reply pylabrobot/liquid_handling/backends/hamilton/STAR.py]: I actually added a bit more

Comment (rickwierenga) [reply pylabrobot/liquid_handling/backends/hamilton/STAR.py]: of course, just making sure you use the correct units :)

**Comment by rickwierenga** [comment]
default liquid height of 1 was actually changed to 0 in #205

https://github.com/PyLabRobot/pylabrobot/pull/205#issuecomment-2288486684

can i merge plr:main into this branch jkh/add-no-lld? needed to get tests to run

**Comment by jkhales** [comment]
> default liquid height of 1 was actually changed to 0 in #205
> 
> [#205 (comment)](https://github.com/PyLabRobot/pylabrobot/pull/205#issuecomment-2288486684)
> 
> can i merge plr:main into this branch jkh/add-no-lld? needed to get tests to run

yea, will let you take over!

**Comment by jkhales** [comment]
bumping this

**Comment by rickwierenga** [comment]
thx for bump & fixing this

**Pull request merged**